src/app/panels/tab-proxy.js

```diff
@@ -96,7 +97,6 @@ export class TabProxy {
       const handlers = Object.keys(this._handlers)
       let i = handlers.indexOf(active)
       if (i >= 0) {
-        i = handlers[i - 1] ? i - 1 : handlers.length - 1
```

Comment:
this line should not be removed cause it completely changes the behavior of this function. then it is no longer `switching to Previous Tab`